pages/common/pixiecore.md

```diff
@@ -0,0 +1,8 @@
+# pixiecore
+
+> A tool to manage network booting of machines.
```

Comment:
Perhaps this will read better?
```suggestion
> Tool to manage booting machines from the network.
```

Reply:
I don't know which one is more accurate, but I think the second variant is more confusing, since it sounds like you manage "booting machines" which are in the network. (Or is this indented?)

Reply:
Ah, ok. What about this then?
```suggestion
> Tool to manage network booting of machines.
```

Reply:
Sounds more tldr-ish 👍🏽